### PR TITLE
BZ 1198792

### PR DESCRIPTION
**Pull request by sshveta**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit c472e7597946228be6d78b7e66e85d0b58346359:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 50389c7d677781934e9bd2c747ff3cd8a6a4a7d1:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 7c2e4f4708aaa00f3bfb33b9a38acf956e5dc494:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d35f9597205236868a9ac824d0193fffd44524a5:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0cf2218934524250c4640d194c4fd24cb7394c36:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/automate/service_dialogs.py [hunk 1]
```diff
@@ -136,6 +138,18 @@ def element_type(self, each_element):
                                 'entry_description': "entry_desc"})
         if choose_type == "Text Area Box":
             text_area_table.click_cell(1, value="Default text")
+        if choose_type == "Drop Down Dynamic List":
+            node1 = each_element.get("field_entry_point")
+            sel.click(element_form.field_entry_point)
+            tree = Tree({
```
Comment (seandst) [inline cfme/automate/service_dialogs.py]: Please define this Tree up above with the other locators, rather than inside this function.

**Comment by dajoRH** [comment]
Lint report for commit c3a850324b86dd28031efc3b350f1e2444823544:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b4cd91dfbbf9b25d282a486ac496d812fb2625ae:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 66cf7c769c1d237c4c27cc85b5b8dace9d343667:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6c9569ec31169a6e9450337cc7d160da843488ad:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 4e8885df326a969e24364e017e97d890ec07e2a7:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/automate/explorer.py [hunk 2]
```diff
@@ -266,7 +267,7 @@ def is_enabled(self):
         return self.enabled
 
     @classproperty
-    def default(cls):
+    def default(self, cls):
```
Comment (seandst) [inline cfme/automate/explorer.py]: A `@classproperty`'s signature should be `def propertyname(cls)`, why the "self"? I'd expect this to TypeError now.

Comment (sshveta) [inline cfme/automate/explorer.py]: Linting was complaining to add self ..

**Comment by dajoRH** [comment]
Lint report for commit 469b4e9500024f3ec1b02c968a2c0a0dafd4f95d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 72f4af78f22dd4a1c5ba8c466aaface5cb0be14d:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**